fe/fe-core/src/main/java/org/apache/doris/system/Backend.java

```diff
@@ -823,4 +826,15 @@ public String getTagMapString() {
         return "{" + new PrintableMap<>(tagMap, ":", true, false).toString() + "}";
     }
 
+    public boolean getPublishTaskLastTimeAccumulated() {
```

Comment:
maybe just return the num, not boolean here ?  because in diagnose SQL it also need the num.

Reply:
fixed